src/hooks/useModal.tsx

```
@@ -0,0 +1,36 @@
+import React, { useContext, createContext, useState, ElementType } from 'react'
+
+type ModalsType = {
+  Component: ElementType
+  props?: any
+}
+
+type ModalContextType = {
+  openModal: (Component: ElementType, props?: {}) => void
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)